torch/distributed/distributed_c10d.py

```diff
@@ -4943,7 +4943,7 @@ def _find_or_create_pg_by_ranks_and_tag(
                 my_ranks = rank_set
         assert my_ranks is not None, "rankset doesn't include the current node"
 
-    my_ranks.sort()
```

Comment:
any difference between these two calls?

Reply:
Oh, yes, the original one is not `torch.compiler()` friendly.